src/components/MovieDetails/MovieDetails.js

```diff
@@ -8,57 +8,64 @@ class MovieDetails extends Component {
-
+            .catch(() => this.setState({ error: 'Having trouble finding this movie right now...please try again.'} ));
     }
 
-    // formatOverviewLength = () => {
-    //     if (this.props.movieInfo.overview.length > 182) {
-    //         const shortOverview = this.props.movieInfo.overview.slice(0, 182) + '...';
+    // formatOverviewLength = (overview) => {
+    //     if (overview.length && overview.length > 20) {
+    //         const shortOverview = overview.slice(0, 20) + '...';
+    //         console.log('Shorty', shortOverview)
     //         return shortOverview;
     //     } else {
-    //         return this.props.movieInfo.overview;
+    //         console.log('OVERVIEW', overview)
+    //         return overview;
     //     }
     // }
+   
     render() {
-        // const formattedDate = 
-        console.log("DO YOU RENDER")
-        const movieInfo = this.props.movieInfo
-        // !this.state.movieInfo.id ? movieInfo = this.props.movieInfo : { movieInfo } = this.state.movieInfo;
-        console.log('movie info in render', movieInfo)
-        const { resetClickedMovie } = this.props;
-        const formattedRating = movieInfo.average_rating.toFixed(0);
+        
+        if (this.state.movieInfo.id) {
+            const { movieInfo } = this.state
+            console.log("WORKING?", movieInfo)
+        }
 
-        return (
+            return (
+    
+                <section className="movieDetailsContainer" style={{ backgroundImage: `url(${this.state.movieInfo.backdrop_path})`}}>
+                    <Link to="/">
+                        <button className='returnHome'>X</button>
+                    </Link>
+                    {this.state.error && <h3>{this.state.error}</h3>}
+                    {!this.state.movieInfo && !this.state.error && <h2 className='loading-message'>Page Loading 🍿</h2>}
 
-            <section className="movieDetailsContainer" style={{ backgroundImage: `url(${movieInfo.backdrop_path})`}}>
-                <Link to="/">
-                    <button className='returnHome' onClick={() => {resetClickedMovie()}}>X</button>
-                </Link>
-                <div className='movieDescription'>
-                    <img className='movie-poster' src={movieInfo.poster_path} id={movieInfo.id} alt={'Cover art image for ' + movieInfo.title}></img>
-                    <div className='movieDetailsCard'>
-                        <h2>{movieInfo.title}</h2>
-                        <h4>{movieInfo.tagline}</h4>
-                        <p>{"Genre: " + movieInfo.genres}</p>
-                        <p>{"Rating: " + formattedRating + "/10"}</p>
-                        <p>{movieInfo.release_date}</p>
-                        {/* <p className='overview'>{this.formatOverviewLength()}</p> */}
-                    </div>
-                </div>
-            </section>
-        )
-    }
+                    {this.state.movieInfo.id && !this.state.error &&
```

Comment:
We should research a way of destructuring here so we aren't having to use this.state.movieInfo throughout this render method